### PR TITLE
Add `Donut Cup` wiki article

### PR DESCRIPTION
**Pull request by Nivalyx**
wiki article for the currently ongoing [Donut Cup](https://osu.ppy.sh/community/forums/topics/1910425).

for [Rentai](http://osu.ppy.sh/u/11033243) and [KKipalt](http://osu.ppy.sh/u/6889573) 🐇 

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by Nivalyx** [comment]
a few things to note :

- yes the tournament offers actual donuts as their prizes
- yes the tournament refers to their finals and grand finals stage as "friend" and "girl friend" interchangeably

image
image

- there was a discussion

File: wiki/Tournaments/Donut_Cup/1/en.md [hunk 1]
```diff
@@ -0,0 +1,593 @@
...
+| Quarterfinals | 2024-06-17/2024-06-23 |
+| Semifinals | 2024-06-24/2024-06-30 |
+| Finals[^stage-note-1] | 2024-07-01/2024-07-07 |
+| Grand Finals[^stage-note-2] | 2024-07-08/2024-07-15 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 1 month of osu!supporter for each team member, custom-made profile banner, a batch of freshly-baked donuts (conditional according to feasibility of the delivery) |
```
Comment (Walavouchey) [inline wiki/Tournaments/Donut_Cup/1/en.md:31]:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 1 month of osu!supporter for each team member, custom-made profile banner, a batch of freshly baked donuts (conditional according to feasibility of the delivery) |
```

File: wiki/Tournaments/Donut_Cup/1/en.md [hunk 2]
```diff
@@ -0,0 +1,593 @@
...
+
+### Match regulations
+
+1. Prior to starting the match, representatives from each team must run the `!roll` command in order to determine the banning and picking order.
+   - The winner of the `!roll` gets to determine who gets the first pick and the second ban.
+   - The loser of the `!roll` gets the opposite by default.
+   - This rule does not apply in the Qualifier lobbies.
+2. Each team has to ban **two beatmaps** from the corresponding mappool. These beatmaps will not be allowed to be picked by any team during the entire match.
+   - The bans will be done alternatingly (A–B–A–B) between the two teams.
+   - Barring the tiebreaker, there are no restrictions as to which maps may and may not be banned in a match.
```
Comment (Walavouchey) [inline wiki/Tournaments/Donut_Cup/1/en.md:578]: more natural

```suggestion
   - Barring the tiebreaker, there are no restrictions as to which maps may or may not be banned in a match.
```

File: wiki/Tournaments/Donut_Cup/1/en.md [hunk 3]
```diff
@@ -0,0 +1,593 @@
...
+3. Each team will be given a chance to pick **one warm-up beatmap** (up to 4:30 max) to be played in the lobby.
+   - Playing a warm-up beatmap is not mandatory, and teams may elect to skip their warm-up pick should they wish to.
+4. Teams are expected to exercise common sense in pick time windows.
+   - If a team is unable to come up with a pick within a 90–second time window of their picking turn, the pick will be given to the other team.
```
Comment (Walavouchey) [inline wiki/Tournaments/Donut_Cup/1/en.md:582]:
```suggestion
   - If a team is unable to come up with a pick within a 90-second time window of their picking turn, the pick will be given to the other team.
```

File: wiki/Tournaments/Donut_Cup/1/en.md [hunk 4]
```diff
@@ -0,0 +1,593 @@
...
+   - This rule does not apply in the Qualifier lobbies.
+2. Each team has to ban **two beatmaps** from the corresponding mappool. These beatmaps will not be allowed to be picked by any team during the entire match.
+   - The bans will be done alternatingly (A–B–A–B) between the two teams.
+   - Barring the tiebreaker, there are no restrictions as to which maps may and may not be banned in a match.
+3. Each team will be given a chance to pick **one warm-up beatmap** (up to 4:30 max) to be played in the lobby.
+   - Playing a warm-up beatmap is not mandatory, and teams may elect to skip their warm-up pick should they wish to.
+4. Teams are expected to exercise common sense in pick time windows.
+   - If a team is unable to come up with a pick within a 90–second time window of their picking turn, the pick will be given to the other team.    
+5. In a Free Mod pick, each team has to field **at least two different mod combinations** to play the beatmap with.
+   - No Mod doesn't count as an unique mod combination.
```
Comment (Walavouchey) [inline wiki/Tournaments/Donut_Cup/1/en.md:584]:
```suggestion
   - No Mod does not count as an unique mod combination.
```

File: wiki/Tournaments/Donut_Cup/1/en.md [hunk 5]
```diff
@@ -0,0 +1,593 @@
...
+   - Hard Rock and Hidden + Hard Rock counts as the same mod combination.
+   - There are no bonus score multipliers for players who are playing with either Easy or Flashlight.
+6. In the case of a tiebreaker, the tiebreaker map will be played with the Free Mod option enabled.
+   - Playing the tiebreaker map with a mod is *not* mandatory.
+7. The results of each match and any other relevant information regarding the match will be noted by the referee after the match has been concluded.
+
```
Comment (Walavouchey) [inline wiki/Tournaments/Donut_Cup/1/en.md:590]:
```suggestion

## Notes

```

File: wiki/Tournaments/Donut_Cup/1/en.md [hunk 6]
```diff
@@ -0,0 +1,593 @@
...
+  3. [AVTechNO! - boku-boku SuketchP Remix (feat. Hatsune Miku) (rollpan) \[Extreme - OTST\]](https://osu.ppy.sh/beatmapsets/1829854#osu/3755660)
+- Double Time
+  1. [Ceui - Hoshi Ori Yume Mirai (Hishiro Chizuru) \[Cyndere's Expert\]](https://osu.ppy.sh/beatmapsets/1517513#osu/3136056)
+  2. [senya - Kanjou Chemistry (Drum 'n' Bass Remix) (Skubi) \[PAJWOJ's Vestige of Days Gone By\]](https://osu.ppy.sh/beatmapsets/2084917#osu/4384262)
+  3. [fripSide - only my railgun -Euro Beat Charger Mix- (Nos\_If) \[Insane\]](https://osu.ppy.sh/beatmapsets/1558910#osu/3184307)
+  4. [Susumu Hirasawa - SWITCHED-ON LOTUS (Starrodkirby86) \[KIRBY Mix Deluxe\]](https://osu.ppy.sh/beatmapsets/16457#osu/58970)
+- Free Mod
+  1. [Tokyo Jihen - Jusui Negai -The Suicide- (seros) \[Kyouiku\]](https://osu.ppy.sh/beatmapsets/1761649#osu/3605664)
+  2. [Tsukifune Kosho-do - Tabi no Owari to Shukusai no Yoru (vergil chair) \[Journey\]](https://osu.ppy.sh/beatmapsets/1346576#osu/2788659)
+  3. [paraoka - rampage (kiri09) \[9\]](https://osu.ppy.sh/beatmapsets/113359#osu/294086)
```
Comment (Walavouchey) [inline wiki/Tournaments/Donut_Cup/1/en.md:173]:
```suggestion
  3. [paraoka - rampage (kiri09) \[9\]](https://osu.ppy.sh/beatmapsets/113359#osu/294086)
```

File: wiki/Tournaments/Donut_Cup/1/en.md [hunk 7]
```diff
@@ -0,0 +1,593 @@
...
+  2. [Shpongle - Levitation Nation (piroshki) \[Welcome to... Brazil?!\]](https://osu.ppy.sh/beatmapsets/1801375#osu/3693390)
+  3. [HyuN - Illusion of Inflict (Icekalt) \[N/A's Extra\]](https://osu.ppy.sh/beatmapsets/968923#osu/2109396)
+- Double Time
+  1. [WEAVER - Kuchizuke Diamond (Net0) \[Happy Birthday\]](https://osu.ppy.sh/beatmapsets/1103117#osu/2304350)
+  2. [MORE MORE JUMP! x Hatsune Miku - Idol Shin'etai (xidorn) \[BURST\]](https://osu.ppy.sh/beatmapsets/2195379#osu/4645182)
+  3. [fripSide - flower of bravery (ImpurePug) \[Insane\]](https://osu.ppy.sh/beatmapsets/1824641#osu/3744397)
+  4. [RuLu feat. Youi - Xylophone (Irin) \[Fish Rose's Insane\]](https://osu.ppy.sh/beatmapsets/1352063#osu/2892338)
+- Free Mod
+  1. [the liquid movement - Love sick (mintIceCream\_) \[tv\]](https://osu.ppy.sh/beatmapsets/2138395#osu/4654833)
+  2. [Kikuo - Yousei no Kiri (Vararaup) \[Damnjelly's Magical\]](https://osu.ppy.sh/beatmapsets/1880072#osu/3870421)
```
Comment (Walavouchey) [inline wiki/Tournaments/Donut_Cup/1/en.md:230]:
```suggestion
  2. [Kikuo - Yousei no Kiri (Vararaup) \[Damnjelly's Magical\]](https://osu.ppy.sh/beatmapsets/1880072#osu/3870421)
```

File: wiki/Tournaments/Donut_Cup/1/en.md [hunk 8]
```diff
@@ -0,0 +1,593 @@
...
+  1. [AAAA - Hoshizora no Kanransha (Setu) \[Conclusion\]](https://osu.ppy.sh/beatmapsets/2094949#osu/4393494)
+  2. [AAAA - carnation (Mirash) \[Mirashola Fanboy's Expert\]](https://osu.ppy.sh/beatmapsets/1893490#osu/3934268)
+  3. [midwxst - Smile (feat. glaive) (- ascended -) \[Feign\]](https://osu.ppy.sh/beatmapsets/1458846#osu/2997929)
+- Double Time
+  1. [Kurosaki Maon - Rakuen no Tsubasa (Asagi) \[Insane\]](https://osu.ppy.sh/beatmapsets/227041#osu/529108)
+  2. [Hanazono Mei - Kurenai Fubuki (Nao Tomori) \[Collab Insane\]](https://osu.ppy.sh/beatmapsets/1549881#osu/3167236)
+  3. [lapix - Flamenco House (Luminiscental) \[Insane\]](https://osu.ppy.sh/beatmapsets/1865291#osu/3857255)
+- Free Mod
+  1. [PUP - Factories (Sparxo) \[THROUGH THE NIGHT\]](https://osu.ppy.sh/beatmapsets/1239345#osu/2576707)
+  2. [inabakumori - Tokoshizume  (xyperX) \[Kard's Insane\]](https://osu.ppy.sh/beatmapsets/1777533#osu/3657734)
```
Comment (Walavouchey) [inline wiki/Tournaments/Donut_Cup/1/en.md:283]:
```suggestion
  2. [inabakumori - Tokoshizume (xyperX) \[Kard's Insane\]](https://osu.ppy.sh/beatmapsets/1777533#osu/3657734)
```

File: wiki/Tournaments/Donut_Cup/1/en.md [hunk 9]
```diff
@@ -0,0 +1,593 @@
...
+  4. [AJURIKA - BIG som tam (KKipalt) \[:drool:\]](https://osu.ppy.sh/beatmapsets/1833709#osu/3764497)
+- Hidden
+  1. [yak\_won - Wind Tales (dakiwii) \[catri\]](https://osu.ppy.sh/beatmapsets/1952375#osu/4042950)
+  2. [I love you orchestra - Coconut Love (funny) \[Nucifera\]](https://osu.ppy.sh/beatmapsets/1665501#osu/3400091)
+- Hard Rock
+  1. [Oohashi Takuya (from Sukima Switch) - Katamari on the Wings (melon boy) \[OK MR. SUNSHINE\]](https://osu.ppy.sh/beatmapsets/1867650#osu/3841928)
+  2. [Rabbit House - das Leben nach dem Tode (Rentia) \[Unterwelt\]](https://osu.ppy.sh/beatmapsets/1733348#osu/3542916)
+- Double Time
+  1. [ICHIKO - I SAY YES (N a s y a) \[Insane\]](https://osu.ppy.sh/beatmapsets/2179900#osu/4604218)
+  2. [yanaginagi - Sentimental (defiance) \[Insane\]](https://osu.ppy.sh/beatmapsets/1562788#osu/3191336)
```
Comment (Walavouchey) [inline wiki/Tournaments/Donut_Cup/1/en.md:302]:
```suggestion
  2. [yanaginagi - Sentimental (defiance) \[Insane\]](https://osu.ppy.sh/beatmapsets/1562788#osu/3191336)
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
skimmed, ok afterwards

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**